src/hooks/useActiveProject.test.tsx

```diff
@@ -104,15 +108,15 @@ test('exposes some props from useSubjectProjects and useMe', async () => {
     isFetched: true,
   });
   useMeMock.mockReturnValue({
-    isLoading: true,
```

Comment:
This was actually a pretty good indication that we had a problem here since isLoading and isFetched should be mutually exclusive .